.github/workflows/verify-platform.yml

```diff
@@ -24,7 +24,8 @@ jobs:
        path: 'platform'
        submodules: 'recursive'
        lfs: true
-       repository: 'eclipse-platform/eclipse.platform.releng.aggregator'
+       repository: 'HannesWell/eclipse.platform.releng.aggregator'
```

Comment:
Is this intended?

Reply:
It is an intended change for testing this PR, but I will revert that before merging this.

Reply:
> It is an intended change for testing this PR, but I will revert that before merging this.

Done.